docs/catalog/cloud/containerize-your-workload-where-applicable.md

```diff
@@ -0,0 +1,40 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- cloud
+- compute
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Containerize your workloads
+
+## Description
+Containers may reduce unnecessary resource allocation and increase utilization of the compute resources, as they allow for bin packing and require less compute resources than virtual machines.
+
+## Solution
+Use [Draft](https://github.com/Azure/draft) tool to simplify application containerization by generating Dockerfiles and Kubernetes manifests.
```

Comment:
Draft is a vendor-specific solution. Patterns can reference vendored tools as options or examples, but the pattern solution should be vendor-agnostic.

Reply:
proposal for phrasing:

Containerize your applications with their minimal dependencies. 

You can use [Draft](https://github.com/Azure/draft) tool to simplify application containerization by generating Dockerfiles and Kubernetes manifests.